src/types/FamilyUnitAgesProps.ts

```diff
@@ -0,0 +1,5 @@
+export default interface FamilyUnitAgesProps {
```

Comment:
This should be `under3` ( x < 3), `between3and15` ( 3 <= x <= 15 ) and `over15` ( x > 15), right?